src/main/java/com/raoulvdberge/refinedstorage/gui/grid/sorting/GridSortingInventoryTweaks.java

```diff
@@ -0,0 +1,36 @@
+package com.raoulvdberge.refinedstorage.gui.grid.sorting;
+
+import com.raoulvdberge.refinedstorage.apiimpl.network.node.NetworkNodeGrid;
+import com.raoulvdberge.refinedstorage.gui.grid.stack.GridStackItem;
+import com.raoulvdberge.refinedstorage.gui.grid.stack.IGridStack;
+import invtweaks.api.InvTweaksAPI;
+import net.minecraftforge.fml.common.Loader;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+
+public class GridSortingInventoryTweaks extends GridSorting {
+    InvTweaksAPI api;
+
+    public GridSortingInventoryTweaks() {
+        try {
+            api = (InvTweaksAPI)Class.forName("invtweaks.forge.InvTweaksMod", true, Loader.instance().getModClassLoader()).getField("instance").get(null);
+        } catch (Exception ex) {
+            api = null;
```

Comment:
Unneeded declaration. Please declare `api` field by default as `null` (and private, as the coding style).

You also need a space after your cast to `InvTweaksAPI`.